### DISCUSSION TITLE
Tandoor script ends with postgres error

### DISCUSSION THREAD

**Discussion by Joe-raspi** [tteck/Proxmox#3414] [Help Needed] [answered]
Hi,
I found this site with Proxmox scripts.
Great idea, helpful sripts!
I tried to install tandoor for Ubuntu 22.04 and 24.04.
The script ends with the following message in both cases:

Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

```
The following packages have unmet dependencies:
 - postgresql-16 : Depends: postgresql-client-16 but it is not going to be installed
                 Depends: libicu72 (>= 72.1~rc-1~) but it is not installable
                 Depends: libllvm16 but it is not installable
                 Depends: libzstd1 (>= 1.5.2) but 1.4.8+dfsg-3build1 is to be installed
                 Recommends: sysstat but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

[ERROR] in line 82: exit code 0: while executing command $STD apt-get install -y postgresql-16

```

As I can see Ubuntu 24.04 is using postgres-15.

Joe

**Comment by tteck** [comment, marked as answer]
Why not try the default Linux distribution?

🛑 Failure to comply with the following guideline.
> - If the default Linux distribution is not adhered to, script support will be discontinued.

**Comment by Joe-raspi** [comment]
Hello,

I try to use all my machines/VMs with the same distri. That is Ubuntu because of long time support.
But of course I could use another one.
What distri do you consider "default"?

Joe

**Reply by tteck** [reply]
The script provides that information!

![Screenshot 2024-07-22 12 57 24 PM](https://github.com/user-attachments/assets/7855f32a-d847-42d6-9e83-c0c3737c4cbb)
